aggregator/leader_flush_mgr.go

```diff
@@ -460,9 +495,10 @@ func (h flushMetadataHeap) heapify(i, n int) {
 			smallest = right
 		}
 		if smallest == i {
-			return
+			break
 		}
 		h[i], h[smallest] = h[smallest], h[i]
 		i = smallest
 	}
+	return i > i0
```

Comment:
Add a comment on what's the returned value?

Reply:
Sure updated.